satpy/readers/hdfeos_base.py

```diff
@@ -333,6 +333,10 @@ class HDFEOSGeoReader(HDFEOSBaseFileReader):
         "satellite_zenith_angle": ("SensorZenith", "Sensor_Zenith"),
         "solar_azimuth_angle": ("SolarAzimuth", "SolarAzimuth"),
         "solar_zenith_angle": ("SolarZenith", "Solar_Zenith"),
+        "water_present": "WaterPresent",
+        "land_sea_mask": "Land/SeaMask",
+        "height": "Height",
+        "range": "Range",
```

Comment:
I'm tempted to say these should be assigned via the `file_key` in the YAML definition, but then again the way you've done this isn't disallowed... :man_shrugging:

Reply:
I'm just copying the format of what was already there, tbh!